### PR TITLE
Grafana datasource plugin tutorial + reference docs.

### PR DESCRIPTION
**Pull request by htroisi**

### CODE REVIEW

File: content/reference/05-plugins/01-grafana.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+metaTitle: "Reference | Grafana Datasource Plugin "
+metaDescription: "Pixie's Grafana Datasource Plugin"
+order: 5
+---
+
+The Grafana Pixie Datasource allows you to visualize data from the Pixie observability platform in Grafana.
+
+## Installation
+
+Install the plugin using Grafana's command line tool or using
```
Comment (nserrino) [inline content/reference/05-plugins/01-grafana.md]: fragment?

File: content/reference/05-plugins/01-grafana.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### Macros
+
+Grafana uses macros to add dashboard context to a query. The following macros are supported for PxL queries executed by plugin:
+
+- `__time_from` will be replaced by the start of the currently active time selection. Example usage:
+
+```python
+import px
+df = px.DataFrame(table='http_events', start_time= __time_from())
```
Comment (nserrino) [inline content/reference/05-plugins/01-grafana.md]: I don't think we use the () to invoke?

File: content/reference/05-plugins/01-grafana.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+- Learn more about the PxL language [here](/reference/pxl/).
+- Learn how to write a PxL script [here](/tutorials/pxl-scripts).
+
+<Alert variant="outlined" severity="warning">
+  Note that the plugin does not support Vis Spec input. PxL queries must return a table using the `px.display()` call.
+</Alert>
+
+If there are syntax errors in your PxL query, Grafana will display an error message.
+
+If a query output table has a `time_` column, the plugin will automatically sort the column.
```
Comment (vishjain) [inline content/reference/05-plugins/01-grafana.md]: Actually, if the query output table has a column with a timestamp in general, the plugin will sort the data by time.

File: content/tutorials/08-grafana.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+df.error = df.resp_status >= 400
+
+# Group HTTP events by service, counting errors and total HTTP events.
+df = df.groupby(['service']).agg(
+    error_count=('error', px.sum),
+    total_requests=('resp_status', px.count)
+)
+
+# Output the DataFrame.
+px.display(df)
```
Comment (vishjain) [inline content/tutorials/08-grafana.md:214]: Did you want to include the multiple display calls/multiple tables can be returned or is that extraneous information?

**Review by vishjain — DISMISSED** [review]

File: content/reference/05-plugins/01-grafana.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+- Learn more about the PxL language [here](/reference/pxl/).
+- Learn how to write a PxL script [here](/tutorials/pxl-scripts).
+
+<Alert variant="outlined" severity="warning">
+  Note that the plugin does not support Vis Spec input. PxL queries must return a table using the `px.display()` call.
+</Alert>
+
+If there are syntax errors in your PxL query, Grafana will display an error message.
+
+If an output table has a column containing timestamps, the plugin will sort the table by the timestamp column.
```
Comment (nserrino) [inline content/reference/05-plugins/01-grafana.md]: by the first timestamp column

File: content/tutorials/08-grafana.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+Next, let's add a panel:
+
+2. In the New Dashboard view, click **Add an empty panel**.
+
+3. In the Edit Panel view, go to the **Query** tab.
+
+4. Configure the query by selecting -- `Pixie Grafana Datasource Plugin` -- from the [data source selector](https://grafana.com/docs/grafana/latest/panels/queries/#data-source-selector).
+
+5. Click the **Save** icon in the top right corner of your screen to save the dashboard.
```
Comment (nserrino) [inline content/tutorials/08-grafana.md:62]: can we take a new screenshot without unsigned?

File: content/tutorials/08-grafana.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+The above PxL query outputs a table of timeseries data showing overall HTTP request throughput. Request throughput is calculated by counting the number HTTP requests that Pixie automatically traces in your cluster.
+
+This query uses the `__time_from` and `__interval` macros to add dashboard context to the query. See the full [list of macros](reference/plugins/grafana/#macros) supported by this plugin.
+
+2. On the Panel tab, under the **Visualization** drop-down menu, select **Time series**.
+
+3. Add units to the graph: on the **Field** tab, under the **Standard options** drop-down menu, under the **Unit** menu, select **requests/sec**.
+
+You should see something similar to the following:
```
Comment (nserrino) [inline content/tutorials/08-grafana.md]: You might want to say you should see data plotted over time, and the exact numbers will vary depending on the traffic in your cluster

File: content/tutorials/08-grafana.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+px.display(per_ns_df['time_', 'service', 'request_throughput'])
+```
+
+This PxL query outputs a table of timeseries data for HTTP request throughput per service
+
+3. On the Panel tab, under the **Visualization** drop-down menu, select **Time series**.
+
+4. Add units to the graph on the **Field** tab, under the **Standard options** drop-down menu, under the **Unit** menu, select **requests/sec**.
+
+You should see something similar to the following:
```
Comment (nserrino) [inline content/tutorials/08-grafana.md]: same as above

File: content/tutorials/08-grafana.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+
+# Output the DataFrame.
+px.display(df)
+```
+
+3. On the Panel tab, under the **Visualization** drop-down menu, select **Bar chart**.
+
+4. Flip the bar chart orientation to horizontal to more easily read the service names. On the **Field** tab, under the **Display** drop-down menu, select **Orientation** > **Horizontal**.
+
+You should see something similar to the following:
```
Comment (nserrino) [inline content/tutorials/08-grafana.md]: one more time

**Review by nserrino — DISMISSED** [review]
some feedback left but overall looks good.

**Review by nserrino — APPROVED** [review]

**Review by aimichelle — APPROVED** [review]

**Pull request merged**